apps/video/js/video_utils.js

```diff
@@ -1,3 +1,6 @@
+'use strict';
+
+/* jshint unused: false */
```

Comment:
You may consider using ```/* exported VideoUtils */``` instead of the unused declaration. It's a little bit more semantically correct.